### ISSUE TITLE
Download LUA script from specific branch

### ISSUE THREAD

**Issue by lesharodin** [ExpressLRS/ExpressLRS-Configurator#128]
Currently when we chose master branch or 2.0RC1 release the lua download link still goes to v1 lua script.
Pls fix :)

**Comment by basters** [comment]
Just tested on configurator version 1.2.3 and it gives me [v2 version](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS/master/src/lua/elrsV2.lua) of script.

Can you please be sure that you use latest version of configurator and provide steps for reproducing your case?

**Comment by lesharodin** [comment]
> Just tested on configurator version 1.2.3 and it gives me [v2 version](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS/master/src/lua/elrsV2.lua) of script.
> 
> Can you please be sure that you use latest version of configurator and provide steps for reproducing your case?

Yes I'm on the latest version. 
Im chosing HM ES24TX and downloading script. There is 3 scripts that I downloaded via configurator (1.2 / 2.0 / master):
[LUAS.zip](https://github.com/ExpressLRS/ExpressLRS-Configurator/files/7517816/LUAS.zip)

![image](https://user-images.githubusercontent.com/12091173/141239148-9daf23b4-24f2-4197-bcf0-ed3fe565868f.png)


They have slightly different sizes, but all of them are 1.0 
![image](https://user-images.githubusercontent.com/12091173/141239047-68336dae-59f0-408d-8dac-cc977a61ad73.png)
The v2 script from github repo is much bigger.
And yes I tested them on a radio, and they definately v1 scripts
